vktrace/vktrace_replay/vkreplay_objmapper_class_defs.h

```diff
@@ -67,7 +67,7 @@ class gpuMemory {
         PageGuardChangedBlockInfo *pChangedInfoArray = (PageGuardChangedBlockInfo *)pSrcData;
         if (pChangedInfoArray[0].length) {
             PBYTE pChangedData = (PBYTE)(pSrcData) + sizeof(PageGuardChangedBlockInfo) * (pChangedInfoArray[0].offset + 1);
-            DWORD CurrentOffset = 0;
+            size_t CurrentOffset = 0;
             for (DWORD i = 0; i < pChangedInfoArray[0].offset; i++) {
                 if ((size_t)pChangedInfoArray[i + 1].length) {
```

Comment:
While you are here, you can remove three `(size_t)` casts from this line and the two that follow.  They are not needed since  `.length` and `.offset` are both `size_t`.  Casts suggest purposeful conversions, and putting them here can be confusing.